plugins/modules/floating_ip.py

```diff
@@ -15,9 +15,8 @@
 description:
   - Create, assign and delete floating IPs on the cloudscale.ch IaaS service.
 notes:
-  - To create a new floating IP at least the C(ip_version) and C(server) options are required.
-  - Once a floating_ip is created all parameters except C(server) are read-only.
-  - It's not possible to request a floating IP without associating it with a server at the same time.
+  - Idempotency depends on the C(ip) parameter.
+  - Once a floating_ip is created all parameters except C(server), C(reverse_ptr) and C(tags) are read-only.
```

Comment:
I think this is missing two commas:

    Once a floating_ip is created, all parameters except C(server), C(reverse_ptr), and C(tags) are read-only.
                                ☝️                                               ☝️